PhysicsTools/PatAlgos/python/slimming/modifiedElectrons_cfi.py

```diff
@@ -0,0 +1,7 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
My knowledge of the unscheduled is that just a dangling module definition doesn't make that module running.
Modules showing up in paths, output "keep" statements and in inputTags in corresponding modules will be running.
(but maybe I'm wrong about this)